.travis.yml

```diff
@@ -24,15 +24,15 @@ env:
     - PHP_VERSION="7.2" NODE_VERSION="10" PRODUCT_VERSION="^2.5" TEST_CMD="bin/behat -v --profile=adminui --suite=richtext"
     - PHP_VERSION="7.2" NODE_VERSION="12" PRODUCT_VERSION="^2.5" TEST_CMD="bin/behat -v --profile=adminui --suite=richtext"
     - PHP_VERSION="7.2" NODE_VERSION="14" PRODUCT_VERSION="^2.5" TEST_CMD="bin/behat -v --profile=adminui --suite=richtext"
-    - PHP_VERSION="7.3" NODE_VERSION="10" TEST_CMD="vendor/bin/behat -v --profile=adminui --suite=richtext --config=behat_ibexa_oss.yaml"
-    - PHP_VERSION="7.3" NODE_VERSION="12" TEST_CMD="vendor/bin/behat -v --profile=adminui --suite=richtext --config=behat_ibexa_oss.yaml"
-    - PHP_VERSION="7.3" NODE_VERSION="14" TEST_CMD="vendor/bin/behat -v --profile=adminui --suite=richtext --config=behat_ibexa_oss.yaml"
-    - PHP_VERSION="7.4" NODE_VERSION="10" TEST_CMD="vendor/bin/behat -v --profile=adminui --suite=richtext --config=behat_ibexa_oss.yaml"
-    - PHP_VERSION="7.4" NODE_VERSION="12" TEST_CMD="vendor/bin/behat -v --profile=adminui --suite=richtext --config=behat_ibexa_oss.yaml"
-    - PHP_VERSION="7.4" NODE_VERSION="14" TEST_CMD="vendor/bin/behat -v --profile=adminui --suite=richtext --config=behat_ibexa_oss.yaml"
+    - PHP_VERSION="7.3" NODE_VERSION="10" TEST_CMD="vendor/bin/behat -v --profile=browser --suite=admin-ui --tags=@richtext --config=behat_ibexa_oss.yaml"
```

Comment:
The latest release contains the Behat tests refactoring, this has to be adjusted